include/jemalloc/internal/lockedint.h

```diff
@@ -80,6 +80,64 @@ locked_dec_u64(tsdn_t *tsdn, malloc_mutex_t *mtx, locked_u64_t *p,
 #endif
 }
 
+static inline bool
```

Comment:
Let's comment on the return value, since it's not completely obvious from the name.